From 646b2218353052dd6d8eb1cc495107a064c026a7 Mon Sep 17 00:00:00 2001
Message-Id: <646b2218353052dd6d8eb1cc495107a064c026a7.1426116068.git.feitong.yi@intel.com>
In-Reply-To: <581bcb8c8802c2b763f5abc462a37e22e0195720.1426116068.git.feitong.yi@intel.com>
References: <581bcb8c8802c2b763f5abc462a37e22e0195720.1426116068.git.feitong.yi@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Mon, 23 Feb 2015 19:54:25 +0000
Subject: [PATCH 5/5] SQUASHME! [VPG]: drm/i915: Fix i915 compilation as a
 module

intel_sync.c wasn't being included in modular builds.

The fumble was introduced in:
commit a9e37dd46bdab147cf6c85e72da6c1a8c64e241d
Author: Tvrtko Ursulin <tvrtko.ursulin@intel.com>
Date:   Fri Dec 20 10:45:02 2013 +0000

    REVERTME [VPG]: drm/i915: Adding HW Native Sync support

Issue: GMINL-6529
Change-Id: I657309ef58ba4cfb36ad3953e25eb4a13d84fbf8
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/Makefile b/drivers/gpu/drm/i915/Makefile
index 94d2039..2f453c8 100644
--- a/drivers/gpu/drm/i915/Makefile
+++ b/drivers/gpu/drm/i915/Makefile
@@ -89,9 +89,9 @@ i915-y += i915_dma.o \
 # with the upstream Linux
 i915-y += i915_ext_ioctl.o
 i915-y += i915_gem_userdata.o
+i915-$(CONFIG_DRM_I915_SYNC)  += intel_sync.o
 
 obj-$(CONFIG_DRM_I915)  += i915.o
 
-obj-$(CONFIG_DRM_I915_SYNC)  += intel_sync.o
 
 CFLAGS_i915_trace_points.o := -I$(src)
-- 
1.7.9.5

